Modules/Core/Common/include/itkObject.h

```diff
@@ -202,6 +202,8 @@ class ITKCommon_EXPORT Object : public LightObject
   /** Remove the observer with this tag value. */
   void
   RemoveObserver(unsigned long tag);
+  void
+  RemoveObserver(unsigned long tag) const;
```

Comment:
Instead of adding an extra RemoveObserver overload, can't we simply add a `const` keyword to the existing RemoveObserver member function?

Reply:
I think AddObserver has a non-const overload, just for historical reasons. For RemoveObserver, it should be fine to just a const-keyword.

Reply:
ah, just looked into the link you mentioned above: https://github.com/InsightSoftwareConsortium/ITK/commit/f2783a9aa0157b50e272545edbcad9c3029195ff#diff-0d6ffb294d56f361c4f3b5f07a0aa40f4e046e3049cb107fe009a2b16b3c1bf6R154-R158 . Thanks for sharing!

Reply:
I was just surprised these are not virtual methods, as they are not it should not matter if any derived class overrode the method.

Reply:
This conversation is also resolved now that you removed the non-const overload. 👍

Reply:
@blowekamp _If_ `RemoveObserver` would have been `virtual`, adding a `const` keyword would have been a possibly breaking API change. But now that it's not virtual, adding this `const` wouldn't be a problem. That's what you mean, right?